### PR TITLE
Bare Metal Forms and Helpers

### PR DESCRIPTION
**Pull request by teekaytech**
## We've successfully completed the Bare Metal Form and Helpers project in the rails module of the technical curriculum.
Below is the summary of some of the major activities carried out:
- We created a User model/migration with username, email, and password fields.
- We also created a UsersController with index, new, create, edit, and update actions.
- We added the create user form with the regular HTML form elements, Rails form_tag, and form_for helpers.
- We modified our forms to display a list of error messages, in case of any validation error while creating or editing the user data.

Kindly review the project for approval. :bow:

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    # @user = User.new(username: params[:username], email: params[:email], password: params[:password])
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to new_users_path
```
Comment (nikasulo) [inline app/controllers/users_controller.rb]: This should read `new_user_path`

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Great work @teekaytech and @ebeagusamuel 
You've both done an awesome job on this project 🚀  Please consider my one comment below, make the correction and request another review. I wish you the very best and see you at the top 🎖️

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (teekaytech) [reply app/controllers/users_controller.rb]: Thank you, for your awesome reply.
The correction has been made. Kindly recheck for approval.

**Comment by teekaytech** [comment]
## Thank you for the review.
The above issue has been fixed.
Kindly re-check for approval.

**Review by nikasulo — APPROVED** [review]
Great work! @teekaytech and @ebeagusamuel 🚀 
Please merge and keep moving forward, I wish you the very best and see you at the top 🎖️

**Pull request merged**